ddtrace/contrib/unittest/patch.py

```diff
@@ -415,6 +504,22 @@ def add_xpass_test_wrapper(func, instance, args: tuple, kwargs: dict):
     return func(*args, **kwargs)
 
 
+def _mark_test_as_unskippable(obj):
+    test_name = obj.__name__
+    test_suite_name = str(obj).split(".")[0].split()[1]
+    test_module_path = os.path.relpath(obj.__code__.co_filename)
+    test_module_suite_name = _generate_module_suite_test_path(test_module_path, test_suite_name, test_name)
+    _initialize_unittest_data()
```

Comment:
Is this necessary because the `skipIf` decorator's actually called when `unittest` is discovering tests?

Also, minor nit: if you're going to call `_initialize_unittest_data()` it no matter what, I'd move it to the top of the function.